### PR TITLE
Update Send Screen for Demo

### PR DESCRIPTION
**Pull request by as-dr**
This updates the Send Screen so it's ready for the demo.

### CODE REVIEW

File: burner-ui/src/Modals/Send/Send.tsx [hunk 1]
```diff
@@ -372,14 +373,14 @@ class SendModal extends Component<SendPageProps, SendPageState, BurnerContext> {
                       >
                         Max
                       </MaxButton>
-                      {/* <AssetSelector
+                      <AssetSelector
                         selected={asset}
                         assets={assets}
                         // onChange={() => this.setState({ asset: newAsset })}
```
Comment (dmihal) [inline burner-ui/src/Modals/Send/Send.tsx]: This should be uncommented, right?

Comment (as-dr) [reply burner-ui/src/Modals/Send/Send.tsx]: I don't see that on my version of the file...

Comment (as-dr) [reply burner-ui/src/Modals/Send/Send.tsx]: This is how I see it: 

```  <AssetSelector
                        selected={asset}
                        assets={assets}
                        // onChange={() => this.setState({ asset: newAsset })}
                        disabled={sending}
                      />```

No errant `{/*`

Comment (dmihal) [reply burner-ui/src/Modals/Send/Send.tsx]: Was talking about the onChange line

**Pull request merged**